keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1376,7 +1377,7 @@ authorization attributes.
 
 Example expressions using `authorizer`:
 
-- `authorizer.resource('signers', 'certificates.k8s.io', '*').name(oldObject.spec.signerName).check('approve').allowed()`
+- `authorizer.resource('signers').group('certificates.k8s.io', '*').name(oldObject.spec.signerName).check('approve').allowed()`
```

Comment:
```suggestion
- `authorizer.resource('signers').group('certificates.k8s.io').name(oldObject.spec.signerName).check('approve').allowed()`
```
(drop the version)

Minor nit: putting resource before group looks backwards to me. Unless we go with my fully-qualified resource name suggestion above, I'd prefer:
```suggestion
- `authorizer.group('certificates.k8s.io').resource('signers').name(oldObject.spec.signerName).check('approve').allowed()`
```

Reply:
I agree `group(...).resource(...)`... is a more natural ordering. You can make both work by defining group and resource for both `Authorizer` (returning a new `ResourceCheck`) and `ResourceCheck` (returning a chained `ResourceCheck`)

Reply:
> Should this just be a single call that takes the fully qualified resource name instead? E.g. roles.rbac.authorization.k8s.io?

Making the fields match what is in RBAC, SubjectAccessReview, and authorizer calls seems better than concatenating to me